src/store/layout/getters.ts

```diff
@@ -53,5 +56,13 @@ export const getters: GetterTree<LayoutState, RootState> = {
         if (config) return { ...config }
       }
     }
+  },
+
+  getSpecificLayout: (state, getters, rootState, rootGetters): string => {
```

Comment:
This returns the layout name, not the layout, so maybe rename to a more appropriate `getSpecificLayoutName` instead?